x-pack/plugins/profiling/common/runtime_types/range_rt.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import * as t from 'io-ts';
```

Comment:
`io-ts` is a runtime validation library that is also type-safe. We use it in `@kbn/typed-react-router-config`, which is the library that we use for type-safe client-side routing.